Sources/Testing/Running/Configuration.TestFilter.swift

```diff
@@ -178,32 +181,92 @@ extension Configuration.TestFilter {
   ///
   /// Matching tests have had _all_ of the tags in `tags` added to them.
   public init(excludingAllOf tags: some Collection<Tag>) {
-    self.init(tags: tags, anyOf: false, membership: .excluding)
+    self.init(_predicate: .tags(Set(tags), anyOf: false, membership: .excluding))
   }
+}
 
-  /// Initialize this instance to represent a regular expression matched against
-  /// a test's ID.
+// MARK: - Operations
+
+extension Configuration.TestFilter {
+  /// An enumeration which represents filtering logic to be applied to a test
+  /// graph.
+  fileprivate enum Operation {
```

Comment:
Should be `Sendable`.

Reply:
Thanks! I expected `-require-explicit-sendable` to catch this, but it doesn't seem to have